<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="path1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Указать путь...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sync">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>410</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Синхронизировать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="path2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>330</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Указать путь...</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="itemlistfrom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>260</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 63 8pt &quot;Bookman Old Style&quot;;
background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="itemlistto">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>260</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 63 8pt &quot;Bookman Old Style&quot;;
background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshbutt">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>350</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Обновить
списки</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathName2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>400</y>
      <width>260</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 63 9pt &quot;Bookman Old Style&quot;;</string>
    </property>
    <property name="text">
     <string>Таков путь...</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathName1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>260</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 63 9pt &quot;Bookman Old Style&quot;;
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Таков путь...</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>О нас</string>
    </property>
    <addaction name="By"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="By">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Авторы</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
